server/scripts/2020TaxSeasonDataScript.php

```diff
@@ -0,0 +1,405 @@
+		$thursdayDates = getWeeklyDatesFromRange('2020-01-30', '2020-04-09');
+		foreach ($thursdayDates as $date) {
+			$shiftId = insertShift("$date 17:00:00", "$date 20:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 6, 6, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 6, 6, 100, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 19:00:00", 5, 5, 100, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Anderson Library data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertCenterForPeopleInNeedData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+	$reviewerRoleId = 4;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Center for People in Need data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 5; // Manually obtained from PROD DB
+
+		// Tuesdays
+		$tuesdayDates = getWeeklyDatesFromRange('2020-02-04', '2020-03-24');
+		foreach ($tuesdayDates as $date) {
+			$shiftId = insertShift("$date 11:00:00", "$date 14:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 11:00:00", 8, 8, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 8, 8, 100, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 4, 4, 100, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Center for People in Need data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertLorenEiseleyLibraryData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+	$reviewerRoleId = 4;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Eiseley data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 6; // Manually obtained from PROD DB
+
+		// Thursdays
+		$thursdayDates = getWeeklyDatesFromRange('2020-02-06', '2020-04-09');
+		foreach ($thursdayDates as $date) {
+			$shiftId = insertShift("$date 16:00:00", "$date 19:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 16:00:00", 8, 8, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 8, 8, 100, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 4, 4, 100, 60, $siteId);
+		}
+
+		// Saturdays
+		$saturdayDates = getWeeklyDatesFromRange('2020-02-01', '2020-04-04')

```

Comment:
You're missing a semicolon at the end of this line and in a few other places. You should consider installing the "PHP Intellisense" VSCode extension -- it'll help highlight small syntax errors like this

Reply:
Got Intellisense figured out. Fixed it